test/acceptance/common/index_test.js

```diff
@@ -21,10 +21,42 @@ describe('Acceptance | Common | Index', function () {
           url: '/throw-error',
         });
         expect(response.statusCode).to.equal(StatusCodes.INTERNAL_SERVER_ERROR);
-        expect(response.result).to.not.equal('Some developer-oriented diagnostic message');
-        expect(response.result).to.equal('An error occurred, please try again later');
+        expect(response.result.message).to.equal('An internal server error occurred');
       });
+
+      it('should not make Boom crash on a Scalingo APIError', async function () {
```

Comment:
Ce qui est testé est `build/controllers/github.js`
Le test est `test/acceptance/common/index_test.js`

Je suggère de :
- le déplacer dans `test/acceptance/build/github_test.js`
- le faire à plus bas niveau 
- changer son titre pour ne pas faire référence au framework Boom qui est un détail d'implémentation (la raison peut être indiquée dans le message de commit)

```suggestion
	context('when Scalingo API returns an error', ()=>{
      		it('should throw an error', async function () {
      	});
```